src/EditDistEpochRecordFilter.hpp

```diff
@@ -86,18 +86,59 @@ namespace geopm
                                    double &stable_period_hysteresis,
                                    double &unstable_period_hysteresis);
         private:
+            /// Implements:
+            ///  1. The stable period detector state machine,
+            ///  2. Returns True if the last record passed to m_edpd
+            ///     is considered as an epoch marker. Marked
+            ///     records are expected to be spaced (in number of
+            ///     records) the length of a period plus/minus erroneous
+            ///     records.
+            ///
+            /// STATE MACHINE OPERATION
+            /// -------------------------
+            /// PERIOD_DETECTED state means we are observing a stable period of N and as long as we stay in this
+            /// state we expect to observe an epoch distance of length N plus/minus erroneously added/removed
+            /// calls, which are indicated by the edit distance (score).
+            ///
+            /// The period stability is measured by comparing the period detected with the most current record vs
+            /// detected with the previous. We store the previous in m_last_period. Only periods of length
+            /// >= MIN_DETECTABLE_PERIOD are considered as potentially stable
```

Comment:
"Only periods of length..."  is repeated a few lines down, maybe drop here

Reply:
Commit 55dcff0 fixes this request.